.github/workflows/typescript-publish.yml

```
@@ -0,0 +1,137 @@
+          npm run build  
+      
+          if npm view "@eclipse-che/license-tool@${NEXT_VERSION}" version 2>/dev/null | grep -q "${NEXT_VERSION}"; then
+            echo "[INFO] Version ${NEXT_VERSION} already published on npm, skipping."
+          else
+            npm publish --tag next --access public --provenance
+          fi
+
+      - name: Build and publish release version
+        if: github.event_name == 'workflow_dispatch'
+        env:
+          GITHUB_TOKEN: ${{secrets.secrets.LICENSE_TOOL_RELEASE_GITHUB_TOKEN }}
```

actionlint (1.7.12)

[error] receiver of object dereference "license_tool_release_github_token" must be type of object but got "string"

(expression)